interfaces/insight.ts

```diff
@@ -1,16 +1,20 @@
-import {FiledRequest} from "./filedRequest";
-import {User} from "./user";
 
 export const INSIGHTS_COLLECTION = "insights"
 
 export interface Insight {
     createdBy: string; // userId
     createDate: number;
+    lastUpdated: number;
+    ticketId: string,
     title: string;
-    request: string[]; // filedRequestIds
+    imgs: {src: string, alt: string}[],
+    outcomeNumber: number,
```

Comment:
How are outcomes measured? If we can map insights to outcomes, we can just take the length of the map

Reply:
outcomeNumber was just ported over from the secondary insight interface, I dont have too much of an opinion on whether it stays or not, though Id say fine tuning the field can go in another CR if youre okay with that